intelmq/lib/upgrades.py

```diff
@@ -571,6 +575,22 @@ def v230_feed_changes(defaults, runtime, harmonization, dry_run):
     return messages + ' Remove affected bots yourself.' if messages else changed, defaults, runtime, harmonization
 
 
+def v300_bots_file_removal(defaults, runtime, harmonization, dry_run):
+    """
+    Remove BOTS file
+    """
+    changed = None
+    messages = []
+    bots_file = Path(os.path.join(CONFIG_DIR, "BOTS"))
+    if bots_file.exists():
+        bots_file.unlink()
+        changed = True
+    else:
+        messages.append("Can't find BOTS file, you have to remove it yourself.")
```

Comment:
In case the user already removed the file, the user would see the (error) message here.

But if `BOTS_FILE` does not exist, we do no longer need to remove it. So we can just remove the else-clause I think.